Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -2858,6 +2864,21 @@
 				B3FC3C0626526EFF00DEED9E /* VitalInfo.swift */,
 				B3FC3C0726526F0000DEED9E /* VitalObserver.swift */,
 				B3FC3C322652AE1400DEED9E /* VitalListener.swift */,
+				B3ACF88B2656424F00738CEE /* VitalReader.swift */,
+				B3ACF8B02656634500738CEE /* VitalMemoryReader.swift */,
+			);
+			path = RUMVitals;
+			sourceTree = "<group>";
+		};
+		B3FC3C1226526F4100DEED9E /* RUMVitals */ = {
+			isa = PBXGroup;
+			children = (
+				B3FC3C3B2653A97700DEED9E /* VitalObserverTest.swift */,
+				B3ACF8B92656644000738CEE /* VitalMemoryReaderTest.swift */,
+			);
+			path = RUMVitals;
+			sourceTree = "<group>";
+		};
```

Comment:
There is sth wrong around this lines and it malforms the `pbxproj` file:
```
--- xcodebuild: WARNING: Unable to open project file '/Users/vagrant/git/Datadog/Datadog.xcodeproj' in workspace '/Users/vagrant/git/Datadog.xcworkspace'.
```
You may want to reset changes in `pbxproj`, do rebase and then add `Vital*` files once again.